### ISSUE TITLE
RemoveEmptyFinalizers (CA1821) does not handle expression bodied finalizers

### ISSUE THREAD

**Issue by mavasani** [dotnet/roslyn-analyzers#1804]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

Example: v1.1.0 and v2.6.1

#### Diagnostic ID

Example: [CA1821](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1821-remove-empty-finalizers?view=vs-2017)

#### Repro steps

Source code:
```
    public class Class1 { }

    public class Class2
    {
        private Class1 _field;
        ~Class2() => _field = null;
    }
```

#### Expected behavior
No CA1821

#### Actual behavior
CA1821 is reported

**Comment by mavasani** [comment]
Should be fixed with https://github.com/dotnet/roslyn-analyzers/pull/1960
